qcodes/_version.py

```diff
@@ -1,11 +1,14 @@
 def _get_version() -> str:
-    from pathlib import Path
+    import sys
 
-    import versioningit
+    if sys.version_info >= (3, 9):
+        from importlib.resources import as_file, files
+    else:
+        from importlib_resources import as_file, files
```

Comment:
no need for as_file here
```suggestion
        from importlib.resources import files
    else:
        from importlib_resources import files
```